app/pages/UcsfDiscoveryForm/UcsfDiscoveryForm.tsx

```diff
@@ -52,6 +53,13 @@ const Page = () => {
   const subcategories: SubcategoryRefinement[] =
     useSubcategoriesForCategory(category?.id ?? null) || [];
 
+  const getResourceIcon = () => {
+    const selectedResource = UCSF_RESOURCES.find(
+      (c) => c.slug === selectedResourceSlug
+    );
+    return selectedResource?.icon ?? "";
+  };
```

Comment:
Is it necessary to wrap this in a function, or can we just directly assign the value to a variable? e.g.

```js
const selectedResource = UCSF_RESOURCES.find(...);
const iconName = selectedResource?.icon ?? "";
```